### ISSUE TITLE
error: element.code is not a function 

### ISSUE THREAD

**Issue by dtruel** [summernote/angular-summernote#85]
Hi,

I'm using this directive, and can get the editor to show up just fine.  Unfortunately, I'm getting this error in angular, and ngModel is not updating.  Any ideas why?  As a note, I am using ui-router, and I wonder if that may be causing issues.

```
TypeError: element.code is not a function
    at ngModel.$render (angular-summernote.ng.js:90)
    at Object.ngModelWatch (angular.js:25423)
    at Object.watchExpression (<anonymous>:754:29)
    at Scope.$digest (angular.js:15818)
    at Scope.$delegate.__proto__.$digest (<anonymous>:844:31)
    at Scope.$apply (angular.js:16097)
    at Scope.$delegate.__proto__.$apply (<anonymous>:855:30)
    at bootstrapApply (angular.js:1660)
    at Object.invoke (angular.js:4478)
    at doBootstrap (angular.js:1658)
```

Additional info:

Occurs on line 90 of the file in question.  I did `$().summernote` which returned a function, so I know summernote is initialized.  I also saw that `element` is a jquery object in the debugger.

More:

I tried manually initializing summernote and then running `.code()` this too is undefined.  Looks like it's not an issue with this library.

**Comment by dtruel** [comment]
I quit using the compressed version of summernote, and that fixed it.  I'm using meteor, and I think that may have been the issue.

**Comment by jkerrwilson** [comment]
As a note to future users who may encounter this problem, I also ran into it using summernote v0.7.0 with angular-summernote v0.3.2. This is resolved by downgrading summernote to v0.6.16.

**Comment by outsideris** [comment]
@jkerrwilson Thank you for reporting. I wrote about this in README.md but it seems not clear. I will write more details.
